### ISSUE TITLE
Compatibility Question

### ISSUE THREAD

**Issue by alanef** [danielbachhuber/gutenberg-plugin-compatibility#9]
A plugin that uses custom post types, but has not defined `show_in_rest' => true,`  &  post_type support includes 'editor' will only permit classic editor not Gutenberg for the CPT.

I would consider that a fail - but before I mark a plugin as such can you clarify thoughts.

**Comment by danielbachhuber** [comment]
Yes, I'd agree with this assessment. You can include an explanation of your judgement in the details field.

**Comment by bobbingwide** [comment]
This is a moot point. If you’re performing tests to see if the new block editor is compatible with your site - code and content - then the answer would be that it is compatible since you will be offered the classic editor. Your CPT is not yet ready for the block editor. 
Post types which are not yet ready include ‘media’. 
Analysis of a subset of my sites indicates that only 5% of my content is ready.

**Comment by glebkema** [comment]
> A plugin that uses custom post types, but has not defined show_in_rest' => true, & post_type support includes 'editor' will only permit classic editor not Gutenberg for the CPT.

@danielbachhuber What reason should I choose for this situation? `Some editor UI is missing`?

**Comment by danielbachhuber** [comment]
@glebkema Sure, whichever your best judgement.
